src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+import React from "react";
+import users from "../../api/users";
```

Comment:
The import of `users` from `../../api/users` is unnecessary in this component. According to the checklist: "Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components". You should remove this import as it is not used in the component.